### ISSUE TITLE
Design a logo for Perseverance, OLS's third cohort

### ISSUE THREAD

**Issue by yochannah** [open-life-science/branding#16]
Each OLS Cohort (group of mentored trainees) of OLS has a name. The cohorts are: 

1. Open Seeds
2. Masked Cohort
3. Perseverance (like the mars rover)
4. Kenaz ([a rune for openness](https://runesecrets.com/rune-meanings/kenaz))
5. Hope

➡️ ➡️ ➡️  **This issue is to suggest a logo for Perseverance only.** 

## Image requirements: 

1. Square
2. Vector format (ideally SVG, but any other vector format is fine too).
3. Not too complex - needs to make sense even when it's a small icon. 

To participate - start by making sketches of your idea. Post a if it's on paper, or a screenshot if your earlier ideas are digital. Post your ideas in-thread - **there is no need to make a pull request for this task.**

Help each other and provide feedback on each others' designs.

**Comment by jessjaey** [comment]
@yochannah  found the "Perseverance" cohort interesting because of the word "perseverance"
I did a little research on supporting words for this
- determination ( "You can do it" )
- belief ( "Believe in yourself" )
- Persistence ("Don't give up" )

I did a quick sketch to show how I view this word:![image](https://user-images.githubusercontent.com/105664383/194909835-6199813c-9bb1-4f08-9cfa-c25e475b629d.jpeg)![image](https://user-images.githubusercontent.com/105664383/194909834-6bfa1cb3-1c99-44fb-a259-7853c68e6180.jpeg)


Combination of a spiral leaf

The leaf would represent the belief of a person
The spiral have various meaning: continuous increase, growth , progression 
For me 😀 : 

That little continuous persistence and belief(perseverance), comes growth.


Logo becomes a brand when the story is told.

This is just a rough draft of how I view this cohort

I would need more details to improve further on what I have in mind for the branding

**Comment by Emidowojo** [comment]
![Perseverance](https://user-images.githubusercontent.com/54295353/194933815-2fb04b3f-fe66-4e83-bb04-6157a2e6a760.svg)
This logo is the perfect representation of perseverance, the perseverance of a turtle with a goal of moving from point a to point b. A turtle symbolizes perseverance and steadfastness. In the fable “The Tortoise and the Hare,” despite his slow pace, it is the tortoise who prevails because of his consistency and focus and this is something a cohort of the Open Life Science should represent.

**Comment by Emidowojo** [comment]
> ![Perseverance](https://user-images.githubusercontent.com/54295353/194933815-2fb04b3f-fe66-4e83-bb04-6157a2e6a760.svg) @yochannah here is a logo that is the perfect representation of perseverance, the perseverance of a turtle with a goal of moving from point a to point b. A turtle symbolizes perseverance and steadfastness. In the fable “The Tortoise and the Hare,” despite his slow pace, it is the tortoise who prevails because of his consistency and focus and this is something a cohort of the Open Life Science should represent.

**Comment by jessjaey** [comment]
@Emidowojo nice point of view 💯

**Comment by Emidowojo** [comment]
> 

Love it!

**Comment by Ronnie-Mercy** [comment]
![logo idea for ols](https://user-images.githubusercontent.com/115509229/194967027-fd83d809-14f3-4013-b269-5dfe3edaf1e2.PNG)
Hello @yochannah  here is a simple logo idea showcasing perseverance and and what is to come after perseverance. As I am trying to see beyond the word "perseverance" itself and also what the brand is all about, I used the spiral circle as an example of how everything in life works hand in hand and also how life itself goes round and round nonstop. The spinning circles meet at each end with an arrow and also the bubble-like star in the middle representing science i.e what comes after enduring. Im still learning about branding and I hope this is a good start!

**Comment by niyolynate** [comment]
Hi,
I Would like to work on this issue. am an Outreachy Applicant too.

**Comment by AdeolaAdekoya** [comment]
![perceverance - 2](https://user-images.githubusercontent.com/86787740/194981337-7fa39a2a-0f6b-4934-a944-15655584b9ba.svg)

![IMG_1516](https://user-images.githubusercontent.com/86787740/194981914-bc2bd76c-d0df-420a-8d67-997e129e84f4.jpg)

Everyone, here is my interpretation of the task. This image depicts a man reaching the finish line on top of one mountain and another mountain to emphasize that there was a journey involved in getting there, and the person was able to reach it despite the difficulty.

Here is my sketch and the digital design.

**Comment by AdeolaAdekoya** [comment]
> Hi, I Would like to work on this issue. am an Outreachy Applicant too.

@niyolynate hello Niyo, nice to meet you. You just need to add the interpretation of the task here to contribute.

**Comment by niyolynate** [comment]
Hello Adeola
Thanks allot.

**Comment by niyolynate** [comment]
![sketch 1](https://user-images.githubusercontent.com/98440370/194989052-d8aa361e-67a6-4fa8-a5e3-47c81d0195f9.png)
![Logo 1 Pers](https://user-images.githubusercontent.com/98440370/194989067-979eb193-383f-4470-88d6-6202144a62ad.png)

Hello everyone, 
this is my interpretation of perseverance, when a dry bean is planted it dies first but still grows. there is still hope.

**Comment by niyolynate** [comment]
A cohort studies are a type of research design that follow groups of people over time. A cohort is a set of users grouped together because of a common identifier
![Uploading sketch 1.png…]()
![ko](https://user-images.githubusercontent.com/98440370/195017305-8b025ca6-7403-44c4-a6c0-781d7a7038c6.png)


Having the ability to not quit in the face of adversity allows you to grow and realize your full potential.

**Comment by oneboyfromife** [comment]
Great work guys, let's put the requirements for the logo to heart

**Comment by oneboyfromife** [comment]
And I have a suggestion also, since she mentioned perseverance like the Mars rover why don't we something about the Mars rover in our logo

**Comment by niyolynate** [comment]
# The cohort logo
## Sketch

![Sketch logo](https://user-images.githubusercontent.com/98440370/195022206-a2667b96-55ab-431e-9e68-69b1365a4492.png)


Cohort Logo
![Cohort logo](https://user-images.githubusercontent.com/98440370/195022317-533e27b5-78b4-42b6-a182-1f4a36330eb5.png)

# Perseverance

Cohort Developers with Perseverance - is a slow consistence effort  of a group of developers with a common goal.

**Comment by niyolynate** [comment]
Thanks allot  
@ Oneboyfromife

**Comment by AdeolaAdekoya** [comment]
> And I have a suggestion also, since she mentioned perseverance like the Mars rover why don't we something about the Mars rover in our logo 

That makes sense

**Comment by AdeolaAdekoya** [comment]
> Hello Adeola
> 
> Thanks allot.

You're welcome 😊

**Comment by oneboyfromife** [comment]
> > And I have a suggestion also, since she mentioned perseverance like the Mars rover why don't we something about the Mars rover in our logo
> 
> That makes sense

Thanks

**Comment by iremide1022** [comment]
@niyolynate this looks amazing, the concept is understandable and simple, do you think the black lines can be removed, let’s check the view of that🤗 or what do you think?

**Comment by Mays26** [comment]
Heres my idea of what perserverance could look like
![Perserverance](https://user-images.githubusercontent.com/116025692/197404433-1b99ba63-afa9-4b26-929a-f9ac19853633.PNG)
@yochannah

**Comment by latifatakin** [comment]
Perseverance Logo

My approach to creating a Logo for OLS cohort (Perseverance)
From my research, perseverance can be defined as the continued effort to do or achieve something despite obstacles, difficulties or opposition. 

Next Step is coming up with a sketch of my proposed OLS Perseverance logo.

Hand Gesture; to represent success.

Curved arrows: represents that the path to success might not be straight but present with obstacles and difficulties.

![IMG-1057](https://user-images.githubusercontent.com/102434385/198120049-9ffa89c3-02d9-48e3-9c8b-f58d2131a8f2.jpg)
<img width="100" alt="Perseverance logo 1" src="https://user-images.githubusercontent.com/102434385/198120729-0e4c1962-7159-4ace-abab-f801dbdf7ad8.png">
<img width="100" alt="Perseverance logo" src="https://user-images.githubusercontent.com/102434385/198120799-4f316409-3dec-4957-b975-e1bd9e678741.png">

**Comment by latifatakin** [comment]
Perseverance Logo

My approach to creating a Logo for OLS cohort (Perseverance)
From my research, perseverance can be defined as the continued effort to do or achieve something despite obstacles, difficulties or opposition. 

Next Step is coming up with a sketch of my proposed OLS Perseverance logo.

Hand Gesture; to represent success.
Curved arrows: represents that the path to success might not be straight but present with obstacles and difficulties.
![IMG-1057](https://user-images.githubusercontent.com/102434385/198124809-2b2581c2-b9a5-4974-ad2f-958310c312f4.jpg)

**Comment by niyolynate** [comment]
Yes Dear. @iremide1022 am going to do just that.

**Comment by niyolynate** [comment]
### Perseverance cohort logo of the OLS.
Like a seed planted, dies and rises again is how we aught to persevere through challenges.
![OKI](https://user-images.githubusercontent.com/98440370/198166099-f33a9793-8507-4107-bec0-1ad8d4145ad9.png)
# Perseverance

**Comment by Emmzybrown** [comment]
Thanks @yochannah for this 

For better understanding on how to first create the logo, let’s briefly understand what perseverance is. 

Perseverance means continued effort to do or achieve something despite difficulties, failure, or opposition 

In other to create this logo, I conducted a little research on objects/images that could depect perseverance, colour style and font. 

**Below are images of the sketching processes I took in creating the logo**

![IMG_20221101_112635_250](https://user-images.githubusercontent.com/85748210/199245932-305268c7-8fd6-40fb-9e46-700d26683c6b.jpg)

![IMG_20221101_112651_149](https://user-images.githubusercontent.com/85748210/199246075-0c2f9e91-88ff-418c-965a-7cdd1500dbc8.jpg)

![IMG_20221101_112656_510](https://user-images.githubusercontent.com/85748210/199246185-ce0f1529-1853-4837-8a06-afa3ad17fe4e.jpg)

![IMG_20221101_112700_300](https://user-images.githubusercontent.com/85748210/199246261-2785fb16-5404-4e77-9dff-124fc4d5b5e0.jpg)

**And here is the final logo design created (Logo is in SVG format)** 

![Perseverance 2](https://user-images.githubusercontent.com/85748210/199246511-99ca2d16-ca1a-44a0-8c2b-24b17bb10242.svg)

**Comment by Emmzybrown** [comment]
> Thanks @yochannah for this
> 
> For better understanding on how to first create the logo, let’s briefly understand what perseverance is.
> 
> Perseverance means continued effort to do or achieve something despite difficulties, failure, or opposition
> 
> In other to create this logo, I conducted a little research on objects/images that could depect perseverance, colour style and font.
> 
> **Below are images of the sketching processes I took in creating the logo**
> 
> ![IMG_20221101_112635_250](https://user-images.githubusercontent.com/85748210/199245932-305268c7-8fd6-40fb-9e46-700d26683c6b.jpg)
> 
> ![IMG_20221101_112651_149](https://user-images.githubusercontent.com/85748210/199246075-0c2f9e91-88ff-418c-965a-7cdd1500dbc8.jpg)
> 
> ![IMG_20221101_112656_510](https://user-images.githubusercontent.com/85748210/199246185-ce0f1529-1853-4837-8a06-afa3ad17fe4e.jpg)
> 
> ![IMG_20221101_112700_300](https://user-images.githubusercontent.com/85748210/199246261-2785fb16-5404-4e77-9dff-124fc4d5b5e0.jpg)
> 
> **And here is the final logo design created (Logo is in SVG format)**
> 
> ![Perseverance 2](https://user-images.githubusercontent.com/85748210/199246511-99ca2d16-ca1a-44a0-8c2b-24b17bb10242.svg)

I need your reviews on this please @yochannah @AdeolaAdekoya @latifatakin

**Comment by Preciousutos** [comment]
Here are my samples for the Perseverance cohort logo. So I took another approach to sketching these logos. After researching
perseverance with respect to the mars rover, I discovered that the latest robotic explorer to land on Mars as part of NASA’s ongoing Mars 2020 mission is a rover called Perseverance. The rover is designed to explore the Martian surface looking for signs of past and present life on the planet to contribute to NASA’s Mars Exploration Programme’s science goals.  
![Perseverance cohort](https://user-images.githubusercontent.com/82149094/199572403-f33d2f57-6d51-4014-a81e-0b7b0c8fb47c.jpg)
